platform/broadcom/docker-syncd-brcm/bcmsh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+/usr/bin/socat - UNIX-CONNECT:/var/run/sswsyncd/sswsyncd.socket
```

Comment:
in the drivshell, can we type quit or exit will that exit bcm shell? if yes, then we need to print out warning to warn user certain action will terminate syncd. 

what is the proper way to exit? we should print a banner to that too.